packages/components/src/components/radio-group/radio-group.ts

```diff
@@ -45,28 +44,33 @@ import type SdRadioButton from '../../_components/radio-button/radio-button';
 export default class SdRadioGroup extends SolidElement implements SolidFormControl {
   static dependencies = { 'sd-button-group': SdButtonGroup };
 
-  protected readonly formControlController = new FormControlController(this);
-  private readonly hasSlotController = new HasSlotController(this, 'label', 'error-text');
+  protected readonly formControlController: FormControlController = new FormControlController(this);
   private customValidityMessage = '';
   private validationTimeout: number;
 
   @query('slot:not([name])') defaultSlot: HTMLSlotElement;
   @query('#validation-input') validationInput: HTMLInputElement;
+  @query('#error-message') errorMessage: HTMLDivElement;
 
   @state() private hasButtonGroup = false;
   @state() defaultValue = '';
 
-  /**  A Boolean attribute which, if present, marks the radio valid or invalid. Please note that 'invalid' can only be used in conjunction with 'this.required'.  */
-  @state() private invalid = false;
```

Comment:
This is deprecated with the new logic in form.ts.  We should have a consistent approach to validity.  None of the other components use an 'invalid' property.